src/formatDate.js

```diff
@@ -1,14 +1,77 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const arrayDate = date.split(fromFormat[3]);
```

Comment:
The splitting of the date string is based on `fromFormat[3]`, which assumes a specific separator position. Ensure that the separator is correctly identified and used.